Core/scripts/dirac-install.py

```diff
@@ -1347,320 +1398,547 @@ def installLCGutils( releaseConfig ):
+  logNOTICE("Running externals post install...")
+  checkPlatformAliasLink()
+  return True
 
-  for termpath in [ '/usr/share/terminfo', '/etc/terminfo' ]:
-    if os.path.exists( termpath ):
-      terminfoLocations.append( termpath )
 
-  return ":".join( terminfoLocations )
+def createBashrcForDiracOS():
+  """ Create DIRAC environment setting script for the bash shell
+  """
+
+  proPath = cliParams.targetPath
+  # Now create bashrc at basePath
+  try:
+    bashrcFile = os.path.join(cliParams.targetPath, 'bashrc')
+    if cliParams.useVersionsDir:
+      bashrcFile = os.path.join(cliParams.basePath, 'bashrc')
+      proPath = os.path.join(cliParams.basePath, 'pro')
+    logNOTICE('Creating %s' % bashrcFile)
+    if not os.path.exists(bashrcFile):
+      lines = ['# DIRAC bashrc file, used by service and agent run scripts to set environment',
+               'export PYTHONUNBUFFERED=yes',
+               'export PYTHONOPTIMIZE=x']
+      if 'HOME' in os.environ:
+        lines.append('[ -z "$HOME" ] && export HOME=%s' % os.environ['HOME'])
+
+      # Determining where the CAs are...
+      if 'X509_CERT_DIR' in os.environ:
+        certDir = os.environ['X509_CERT_DIR']
+      else:
+        if os.path.isdir('/etc/grid-security/certificates'):
+          certDir = '/etc/grid-security/certificates'  # Assuming that, if present, it is not empty, and has correct CAs
+        else:
+          # But this will have to be created at some point (dirac-configure)
+          certDir = '%s/etc/grid-security/certificates' % proPath
+      lines.extend(['# CAs path for SSL verification',
+                    'export X509_CERT_DIR=%s' % certDir,
+                    'export SSL_CERT_DIR=%s' % certDir,
+                    'export REQUESTS_CA_BUNDLE=%s' % certDir])
+
+      lines.append('export X509_VOMS_DIR=%s' % os.path.join(proPath, 'etc', 'grid-security', 'vomsdir'))
+      lines.extend(['# Some DIRAC locations',
+                    '[ -z "$DIRAC" ] && export DIRAC=%s' % proPath,
+                    'export DIRACOS=%s/diracos' % cliParams.basePath,
+                    'export DIRACBIN=%s/sbin:$DIRACOS/bin:$DIRACOs/usr/bin' % cliParams.basePath,
+                    'export DIRACSCRIPTS=%s' % os.path.join("$DIRAC", 'scripts'),
+                    'export DIRACLIB=$DIRACOS/lib',
+                    'export TERMINFO=%s' % __getTerminfoLocations(os.path.join("$DIRACOS", 'share', 'terminfo')),
+                    'export RRD_DEFAULT_FONT=%s' % os.path.join("$DIRACOS", 'share', 'rrdtool',
+                                                                'fonts', 'DejaVuSansMono-Roman.ttf')])
+
+      lines.extend(['# Prepend the PYTHONPATH, the LD_LIBRARY_PATH, and the DYLD_LIBRARY_PATH'])
+
+      lines.extend(['( echo $PATH | grep -q $DIRACBIN ) || export PATH=$DIRACBIN:$PATH',
+                    '( echo $PATH | grep -q $DIRACSCRIPTS ) || export PATH=$DIRACSCRIPTS:$PATH',
+                    '#( echo $LD_LIBRARY_PATH | grep -q $DIRACLIB ) || \
```

Comment:
This can go since you redefine it later, it's just a comment anyway

Reply:
I would like to remove all commends if you agree